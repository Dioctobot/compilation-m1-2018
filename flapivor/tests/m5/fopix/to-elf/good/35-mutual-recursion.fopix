def f (x) =
    x + g (x)

def g (x) =
    if x = 0 then 1 else 1 + f (x - 1) end

val main =
    print_int (g (300))