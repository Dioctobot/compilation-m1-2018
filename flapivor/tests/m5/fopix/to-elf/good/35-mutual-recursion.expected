Process exited with status exited(0).
STDOUT:
45151
STDERR:



